src/KafkaFlow.Abstractions/IWorkerDistributionStrategy.cs

```diff
@@ -0,0 +1,23 @@
+namespace KafkaFlow;
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+/// <summary>
+/// An interface used to create a distribution strategy
+/// </summary>
+public interface IWorkerDistributionStrategy
+{
+    /// <summary>
```

Comment:
The Distribution Strategy is not covered by documentation.
I think it's a good idea to have there how to configure this and the default options available.

Reply:
We will create a task to add this documentation afterwards, agreed?

Reply:
By checking the documentation, I found this section about the [Distribution Strategy](https://farfetch.github.io/kafkaflow/docs/guides/consumers/#distribution-strategy). Maybe we can improve it with the latest updates and make it more visible.

Reply:
Issue for the documentation request created: https://github.com/Farfetch/kafkaflow/issues/466

Reply:
This will be part of the v3 release, right?